pkg/volume/glusterfs/glusterfs.go

```diff
@@ -966,6 +969,11 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 		case "volumetype":
 			parseVolumeType = v
 
+		case "volumeoptions":
+			if len(v) != 0 {
+				parseVolumeOptions = v
```

Comment:
This is unfortunate. Could there be a case a wrong option cause a service disruption? Volumes are created and space allocated but unusable due to unsupported options. If this happens, we need options printed in messages and logs.